qemu/tests/pci_bridge.py

```diff
@@ -0,0 +1,286 @@
+import logging
+
+from virttest import error_context
+from virttest import env_process
+from virttest import utils_test
+from virttest import utils_misc
+
+
+def prepare_pci_bridge(test, params, pci_bridge_num):
+    """
+    Prepare pci-bridge for guest
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param pci_bridge_num: VM pci-bridge number
+    """
+    if pci_bridge_num < 1:
+        test.cancel("There should be at least one pci-bridge!")
+
+    params["pci_controllers"] = ""
+    for index in xrange(pci_bridge_num):
+        params["pci_controllers"] += "pci_bridge%d " % index
+        params["type_pci_bridge%d" % index] = "pci-bridge"
+
+
+def prepare_images(test, params, image_num, device_num=0):
+    """
+    Prepare images attached to pci-bridge for guest
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param image_num: VM image number (except image1)
+    :param device_num: Prepared device number
+    """
+    pci_bridges = params.objects("pci_controllers")
+    pci_bridge_num = len(pci_bridges)
```

Comment:
```
pci_bridge_num = int(params.get("pci_bridge_num", 1))
```
```
for index in xrange(pci_bridge_num):
    params["pci_controllers"] += "pci_bridge%d " % index
```
```
pci_bridges = params.objects("pci_controllers")
pci_bridge_num = len(pci_bridges)
```
It is redundant, why not use `pci_bridge_num` directly

Reply:
1. It updates the existed case pci_bridge.block_stress according to new code, no bug ID.
2. IMHO, It's no difference to replace "pci_bridge_num = len(pci_bridges)" with "pci_bridge_num = int(params.get("pci_bridge_num", 1))".
Thanks!